### DISCUSSION TITLE
Filters disappear on Notifications page

### DISCUSSION THREAD

**Discussion by mrghosti3** [community/community#63895] [Mobile] [answered]
After accessing shortcuts and then going to notifications page the filter options do not appear until you reopen the app.

Platform: Android 13
Version: 1.127.0-beta (10145)

**Comment by eliperkins** [comment]
Hey @mrghosti3, thanks for this feedback!

I think we've been working on a similar bug, but this is the first report we've seen of it affecting Notifications. Can I get some more info from you to help debug?

- What device are you using? (e.g., Google Pixel 6)
- Are you able to share a video of the bug with us? This will help our engineers better diagnose what's happening in the UI that's causing these filters to disappear.

Thanks for helping us improve GitHub Mobile!

**Reply by mrghosti3** [reply]
Hello. Yeah, I can give some more info:

> What device are you using?

Asus Zenfone 9

> Are you able to share a video of the bug with us?

Sadly, no. But I can give you steps on what I do:

1. Open the app. Doesn't matter what page you get at first.
2. Next you go to notifications page. When it loads (the first time), I always see the filters belowe the top bar. If it's already open on app start, the next step.
3. Go to any other page (Home, Trending, Profile). Previously I though it was because I went to shortcuts the filters would not show, but apparently you just need to leave that page.
4. Go back to notifications page and there are no filters at the top. Refreshing doesn't help and the only way to reload that page is to close and open the whole app all over again.

**Reply by eliperkins** [reply, marked as answer]
Awesome, thanks for these detailed steps. I've filled an issue within our repo to track this, and I'll work with the engineering team to get it addressed as soon as possible. 🙇
